### ISSUE TITLE
donate: maybe allow unlabled donation -by default

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#217]
my thinking is: given the Add Labels screen - which is very relaxing and easy to use - it would be better to make the act of uploading an image take no thought.. just click the tab and drop the image, without even  having to think about a labels. The wording could be something like "upload an image: default label: <none>". You could still retain the ability to set a default label.

eg some people might happen to have interesting photos , or be in interesting places, and they dont want to spend time labelling on the site... whilst others are stuck inside in winter or whatever with time on their hands :)  the beauty of online collaboration is the combination. Or it could even be the same person , and they're just squeezing the activity into their day at different times.

Second to this you could prioritise serving unlabelled images in the "add labels" mode.

This might also work better for any forthcoming unified mode,

.. and it would also allow submitting images for *label suggestions* .. actively suggest ways to broaden the vocabulary..

**Comment by bbernhard** [comment]
You are right, that definitely makes sense! I think it should be easy to change that.

> Second to this you could prioritise serving unlabelled images in the "add labels" mode.

I have to check again, but I _think_ it should already be the case. But it's possible that the recent extensions broke the functionality..I'll have a look :)

btw: I am also working on a small GUI application which should it make easier to extract frames from a video. It's inspired by your experiments with VLC(not sure if I remember that one correctly, but I think you extracted the frames with VLC, right?). 

Here's a short demo: 

![frames_extractor](https://user-images.githubusercontent.com/5710631/52301172-1a00da80-298a-11e9-9b62-de3f9c4c41f4.gif)


It's still work in progress, but it hopefully gives you a rough idea of what it will look like.
The basic workflow will be: 

* enter your API token into the application once to connect the application with your account
* select the video you want to use
* extract frames every x seconds
* select all the frames you want to upload and press "donate"
* after you press "donate" a dialog will open where you can optionally add one (ore more) labels to the images and assign the images to an existing image collection.

**Comment by dobkeratops** [comment]
> btw: I am also working on a small GUI application which should it make easier to extract frames from a video. It's inspired by your experiments with VLC(not sure if I remember that one correctly, but I think you extracted the frames with VLC, right?).

Interesting. All I did was grabbed them from a videoplayer, it was ok with trackpad scrolling to advance/rewind and the screen grab hotkey - but thumbnails in a GUI tool as you show would be better. Anything to help you pick out varied images (even sampling of traffic footage would have a lot of very same-y frames)
